rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -856,6 +670,8 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
         } yield space.store.toMap
         Await.result(inspectTaskReceiveFirst.runAsync, 3.seconds)
     }
+
+    checkData(receiveFirstResult)(channel, Seq(GString("Success")), mergeRand)
```

Comment:
Did you forget to delete the line below this?

Reply:
Yes, definitely. Thanks

Reply:
Fixed.